server/README.md

```
@@ -132,35 +134,59 @@ func Register(routerInst *router.Router, pool *pgxpool.Pool) {
+9. Adding to the module registry
+10. Swapping e2e tests to the new handler
+11. Removing old code (for migrations)
+12. Preparing for code review
+
+**The sections below provide detailed code examples and architectural context for each layer. Refer to them while following the checklist.**
+
+---
+
+### Package tree structure
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)